armi/physics/neutronics/fissionProductModel/tests/test_fissionProductModel.py

```diff
@@ -29,7 +29,7 @@ class TestFissionProductModel(unittest.TestCase):
     """
 
     def setUp(self):
-        o = buildOperatorOfEmptyHexBlocks()
+        o = buildOperatorOfEmptyHexBlocks({"cycleLength": 365, "burnSteps": 4})
```

Comment:
Is there a reason we are passing this helper function inputs, rather than just changing the helper function?

Reply:
Outdated. This was no longer necessary once the setting defaults were reinstated.